cassovary-core/src/main/scala/com/twitter/cassovary/util/io/FileReader.scala

```diff
@@ -6,15 +6,24 @@ import com.twitter.logging.Logger
 import com.twitter.util.NonFatal
 
 import scala.io.Source
+import java.io.FileInputStream
```

Comment:
to maintain alphabetic order of the imports, please put these new imports above scala.io.Source